pgcli/main.py

```diff
@@ -819,6 +830,7 @@ def get_last_query(self):
         default=False,
         help='Skip intro on startup and goodbye on exit.')
 @click.option('--prompt', help='Prompt format (Default: "\\u@\\h:\\d> ").')
+@click.option('--prompt-dsn', help='Prompt format for connections using DSN aliases (default to same as for --prompt option).')
```

Comment:
Perhaps just say `Default: "\\u@\\h:\\d>`?

Reply:
If you are connecting using -D option, and specify --prompt, but not --prompt-dsn, then your prompt is not equal to "\\u@\\h:\\d>".
If you prefer version with explicit format I will change this, just explaining my point of view :)

Reply:
Well, in this docstring, you're only explaining the default, which is `\\u@\\h:\\d`. No need to make it more complicated by explaining all the use cases.

Reply:
@msztolcman So yes or no? :)

Reply:
@j-bennet Yes and done :)